src/System.Data.SqlClient/src/Interop/LocaleMapperApiHelper.cs

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
As a corefx practice, since this is only for Non-Windows platform, put this file in a Unix specific folder. 
Also this can reside out of the Interop folder. 
There is already an issue to clean up this folder. We don't want to add any content to it, unless its a real interop.